### DISCUSSION TITLE
Building an open source glossary for digital preservation

### DISCUSSION THREAD

**Discussion by anjackson** [digipres/digipres.github.io#41] [General]
Following World Digital Preservation day, a few people seemed keep to set up a shared glossary of digital preservation. There are already a number of glossaries, so the difference here is that it would be community-owned.

Some possible ideas were explored in digipres/awesome-digital-preservation#2, leading to this from @ross-spencer

> I found a pretty cool single page-glossary template and converted some of the brainscape data to work inside it, removing some duplicates and then manually adding lnter-document links ('see also' links):
> 
> * https://ross-spencer.github.io/digipres-glossary/ 
> * Repo: https://github.com/ross-spencer/digipres-glossary/
> * Info: https://glossary.page/template/
> 
> Very happy to pass custody of something like this along, and discuss a way to get some more community input. 

I can help out here and there, but I don't think I'm able to lead this effort. ]

So this discussion is a call for anyone who likes the idea of working on this to make themselves known and provide feedback on what's been proposed so far!

**Comment by ross-spencer** [comment]
Additional context, the [glossary](https://ross-spencer.github.io/digipres-glossary/) itself was created following this prompt: https://github.com/digipres/awesome-digital-preservation/issues/2#issuecomment-1302092701

So this data already sat there in some form or other. 

Here is a spreadsheet that I created from the Brainscape set and then removed duplicates and a few redundancies. There's also now a list of active definitions: https://docs.google.com/spreadsheets/d/1myDAS_2Y1TpZfhqFZ4XsTlK1pa9a7Bio0HeIzy767qc/edit#gid=0

It presents one option out of a few. I'll outline a pathway using this option:

1. The initiative to use this spreadsheet/seed-data needs to be empowered, so ideally it is hosted hereon the digipres org, not `ross-spencer`'s Github. Members would need adding to this organization and that repo to take the strain off Andy.
1. The definitions need working through on at least three levels:
   1. Redundant or unneeded definitions.
   1. Missing definitions.
   1. Improve the text of definitions. 
1. The repository contains sample issues templates: https://github.com/ross-spencer/digipres-glossary/tree/main/.github/ISSUE_TEMPLATE
1. But needs a code-of-conduct, decision-making process, and volunteers to action decisions. 

How to begin: 

1. Bring the glossary over to the digipres org.
2. Accept volunteers to join the organisation.
3. Begin by accepting issues for proposed changes and promoting discussion around those.
4. Begin editing the source data at a determined date in the future, with the other issues still to be resolved, potentially by discussion here, potentially by a google meetup with the volunteers here and whomever wants to be there.
5. Assignees select issues for the repo and then submit pull requests against issues. 
6. Pull requests start getting merged.

# Alternatives

1. Start a glossary from scratch on the digipres org, with many of the issues above still to be resolved, e.g. code of conduct, process.
2. Find an alternative place to have an open glossary, e.g. a wikimedia template for COPTR.
3. Link purely to existing glossaries in the awesome-list and let the open glossary idea find another home. 
4. Investigate wikibase.cloud post invitational period as an open platform capable of delivering a glossary in a linked open data format.

**Reply by anjackson** [reply]
Thanks @ross-spencer. One high-level issue that might be worth clarifying is that I think the intent is to be [descriptive rather than prescriptive](https://en.wikipedia.org/wiki/Dictionary#Prescriptive_vs._descriptive). i.e. this is not an attempt to define a perfect language system for digital preservation, but to document the various different meanings given to words in use in the field.

As an example, I'm thinking of how OAIS takes _Migration_ to mean a very broad concept (_" The transfer of digital information, while intending to preserve it, within the OAIS."_ including media _Refreshment_), but in common usage it's usually referring to format conversion.

**Reply by ross-spencer** [reply]
I anticipate that could be made clear in any introduction and style guide, it looks like those can maybe be added to a todo list for sure! (Providing other folks agree, but +1 here)

**Reply by nkrabben** [reply]
@anjackson is good to point out that it would be most valuable to have documentation of differences instead of trying to find the perfect definition. I'd even appreciate a few definitions that start "There is no commonly agreed meaning of the <filename>".

After poking at some other glossary/dictionary documentation, it would be good to host multiple definitions per terms, tagged according to source, e.g.
```
filename
1. (SAA Glossary) ....
2. (ECMA 107 11.4) ...
```

Doing that would feel very linked open data/wikidata-ish since it's not privileging any source.

**Reply by NaumannKai** [reply]
Hi @ross-spencer @anjackson @nkrabben I totally agree. Our painpoints with terminology are common throughout science. Every time we transcend language boundaries, discipline boundaries or standard boundaries, we run into slightly incompatible language terms. Building an open source glossary will help immensely. Maybe EOSC EDEN could provide workforce for starting up the endeavour and sketch a longer-term business model? Is it helpful to move our efforts closer to other areas of science and technology, like NFDI's ideas? See https://zenodo.org/records/6759325
